deployment/system-config.md

```diff
@@ -100,6 +100,20 @@ Logs event verbosely.
 
 Invokes fluentd without input plugins.
 
+### `with_source_only`
+
+| type | default | version |
+| :--- | :--- | :--- |
+| bool | nil | 1.18.0 |
+
+Not supported on Windows.
+
+Invokes a fluentd only with input plugins.
+
+See [Source Only Mode](source-only-mode.md) for details.
+
+See also [\<source_only_buffer\> section](#less-than-source_only_buffer-greater-than-section).
```

Comment:
Looks for me that the less-than-source_only_buffer-greater-than-section anchor is missing.
So, the browser cannot navigate to the appropriate part of the page...

Reply:
When I preview this in local environment, it shows as following
![image](https://github.com/user-attachments/assets/680c1bc6-6194-4bbe-aeb6-febe2343b34f)

Reply:
https://github.com/fluent/fluentd-docs-gitbook/pull/527#discussion_r1914534584